rust/noosphere/src/ffi/context.rs

```diff
@@ -131,11 +131,18 @@ pub fn ns_sphere_traverse_by_petname(
 ) -> Option<repr_c::Box<NsSphere>> {
     error_out.try_or_initialize(|| {
         let sphere = noosphere.async_runtime().block_on(async {
+            let raw_petnames = petname.to_str();
```

Comment:
Using `"jordan.chris"` is a new way of defining spheres' relationships (in addition to `Vec<String>` and full subtext slashlinks e.g. `"@jordan.chris"`), should be noted in docs